erpnext/hr/report/monthly_attendance_sheet/monthly_attendance_sheet.py

```diff
@@ -57,9 +57,9 @@ def execute(filters=None):
 
 	data = []
 
+	leave_types = frappe.db.sql("""select name from `tabLeave Type`""", as_list=True)
```

Comment:
Avoid using raw SQL query, whenever possible...
Try using:
``` leaves_types = frappe.get_list("Leave Type") ```
It will return something like:
```
[{'name': 'Partial Paid leaves'},
 {'name': 'Compensatory Off'},
 {'name': 'Earned Leaves'},
 {'name': 'Casual Leave'},
 {'name': 'Leave Without Pay'},
 {'name': 'Privilege Leave'},
 {'name': 'Sick Leave'}]
```